docs/README.md

```diff
@@ -18,6 +18,28 @@ The development of this publicly-accessible client is being managed by Caltrans'
 
 The application is accessible to the public at [benefits.calitp.org](https://benefits.calitp.org).
 
+### Adoption by transit agencies
+
+The following California transit agencies have launched Cal-ITP Benefits for their riders, with the following enrollment pathway options:
+
+| Transit agency                                  | Older adults | Agency card | Veterans | Initial agency launch |
+| ----------------------------------------------- | ------------ | ----------- | -------- | --------------------- |
+| **Monterey-Salinas Transit**                    | Live         | Live        | Live     | 5/2021                |
```

Comment:
A small formatting suggestion for styling and legibility: represent single months with two digits rather than one, i.e. 05 rather than 5. All date with read cleanly together.